### PR TITLE
Retain sentence order in summary

### PR DESCRIPTION
**Pull request by krzd**
When creating the summaries, the most 'useful' sentences are used and ordered according to that metric.
However this often results in sentences being in a different order to the original text and causes confusion when reading.
This ensures that the sentences occur in the same order as in the original text.

### CODE REVIEW

File: tests/data/text/cnn_summary.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
+The storm caused some complications and inconveniences, but no major delays or breakdowns.
 Forecasters see mostly smooth sailing into Thanksgiving.
-The forecast has left up in the air the fate of the balloons in Macy's Thanksgiving Day Parade.
 That's good news for people like Latasha Abney, who joined the more than 43 million Americans expected by AAA to travel over the Thanksgiving holiday weekend.
 Though the worst of the storm has passed, winds could still pose a problem.
-The storm caused some complications and inconveniences, but no major delays or breakdowns.
\ No newline at end of file
+The forecast has left up in the air the fate of the balloons in Macy's Thanksgiving Day Parade.
```
Comment (yprez) [inline tests/data/text/cnn_summary.txt]: The tests are still failing for some reason

**Comment by codelucas** [comment]
Fix the branch conflicts and I'll likely accept. Thanks for this PR, good find

**Comment by krzd** [comment]
@codelucas Done, cheers.

**Comment by codelucas** [comment]
Thanks for this!

**Pull request merged**